<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rating</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Review for Selected Item</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="rating_text">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>160</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="comments_text">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>661</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Comments</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addReviewBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>510</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Review</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
